recipes/_lustre_install.rb

```diff
@@ -92,6 +92,20 @@
   kernel_module 'lnet'
 elsif node['platform'] == 'centos'
   Chef::Log.warn("Unsupported version of Centos, #{node['platform_version']}, supported versions are 7.5, 7.6 and 7.7")
+elsif node['platform'] == 'ubuntu'
+
+  # Install Lustre
+  bash "install lustre" do
+    cwd node['cfncluster']['sources_dir']
+    code <<-LUSTREBUILD
+      set -e
+      wget -O - https://fsx-lustre-client-repo-public-keys.s3.amazonaws.com/fsx-ubuntu-public-key.asc | sudo apt-key add -
+      echo "deb https://fsx-lustre-client-repo.s3.amazonaws.com/ubuntu #{node['cfncluster']['ubuntu']['version']} main" > /etc/apt/sources.list.d/fsxlustreclientrepo.list
+      apt-get update
+      apt install -y lustre-client-modules-$(uname -r)
```

Comment:
can we replace this line and the one above with package or apt_package so that we can add retries?